app/build.gradle

```diff
@@ -62,6 +62,9 @@ android {
         buildConfigField "boolean", "EXPERIMENTAL_FEATURES_ENABLED", "true"
         buildConfigField "String", "WELL_KNOWN_URL_SUFFIX", '".well-known/radix.json"'
         buildConfigField "String", "IMAGE_HOST_BASE_URL", '"https://image-service-dev.extratools.works"'
+        buildConfigField "String", "NPS_SURVEY_URL", '"https://dev-wallet-net-promoter-score.radixdlt.com/"'
+        buildConfigField "String", "REFINER_USER_ID", getPropertyByName('REFINER_USER_ID')
```

Comment:
Is `REFINER_USER_ID` value the same for all users? Note, based on our last demo discussion we should attach a random id to each user.